### PR TITLE
Fix multianswer type question

### PR DESCRIPTION
**Pull request by guillogo**
Hi @safatshahin this is the PR for qtype_multipleanswer. Thank you.

### CODE REVIEW

File: question/engine/bank.php [hunk 1]
```diff
@@ -592,8 +592,14 @@ public function get_questions_from_categories_and_tags_with_usage_counts($catego
...
 
-        $sql = 'SELECT q.*,
-                       qc.contextid
+        $sql = 'SELECT q.id, qc.id as category, q.parent, q.name, q.questiontext, q.questiontextformat,
+                       q.generalfeedback, q.generalfeedbackformat, q.defaultmark, q.penalty, q.qtype,
+                       q.length, q.stamp, q.timecreated, q.timemodified,
+                       q.createdby, q.modifiedby, q.idnumber,
+                       qc.contextid,
+                       qv.status,
+                       qv.id as versionid,
```
Comment (safatshahin) [inline question/engine/bank.php:601]: please also add the version number

File: question/engine/bank.php [hunk 2]
```diff
@@ -609,8 +615,14 @@ public function load_for_cache($questionid) {
...
         list($idcondition, $params) = $DB->get_in_or_equal($questionids);
-        $sql = 'SELECT q.*,
-                       qc.contextid
+        $sql = 'SELECT q.id, qc.id as category, q.parent, q.name, q.questiontext, q.questiontextformat,
+                       q.generalfeedback, q.generalfeedbackformat, q.defaultmark, q.penalty, q.qtype,
+                       q.length, q.stamp, q.timecreated, q.timemodified,
+                       q.createdby, q.modifiedby, q.idnumber,
+                       qc.contextid,
+                       qv.status,
+                       qv.id as versionid,
```
Comment (safatshahin) [inline question/engine/bank.php:624]: please also add version number

**Review by safatshahin — COMMENTED** [review]
Hi @guillogo 
Thanks for working on this, just two minor changes to add the version number for the question engine, and it's good to go.

**Comment by guillogo** [comment]
Thank you @safatshahin for reviewing this PR, changes done. Thank you.

**Pull request merged**